src/components/sortableGridList/index.tsx

```diff
@@ -41,14 +42,12 @@ function SortableGridList<T = any>(props: SortableGridListProps<T>) {
   }, [onOrderChange, data]);
 
   const _renderItem = useCallback(({item, index}: ListRenderItemInfo<ItemProps<T>>) => {
-    const lastItemInRow = (index + 1) % numberOfColumns === 0;
-
     return (
       <SortableItem
         key={item.id}
         data={data}
         {...presenter}
-        style={[itemContainerStyle, lastItemInRow && {marginRight: 0}]}
+        style={itemContainerStyle}
```

Comment:
`itemContainerStyle` is not missing (along others) from the dependencies array

Reply:
I added what I think necessary